docs/docker.md

```diff
@@ -76,7 +76,7 @@ To set up a CDash production instance using docker compose, follow these steps:
 docker compose --env-file .env \
 	   -f docker/docker-compose.yml \
 	   -f docker/docker-compose.production.yml \
-	   -f docker/docker-compose.mysql.yml \
+	   -f docker/docker-compose.postgres.yml \
```

Comment:
Should we fold the contents of `docker-compose.postgres.yml` into `docker-compose.yml` now that we only support a single database backend?

Reply:
I thought about doing that, but I think it makes sense to leave it separate in case users want to use docker compose with a separate standalone database, similar to the way we treat minio.  Thoughts?